cmd/snapctl/task.go

```diff
@@ -48,7 +48,9 @@ var (
 	unionParseFormat = timeParseFormat + " " + dateParseFormat
 )
 
+// const defines constants for the length of a string.
 const (
+	// K defines the length of a string.
```

Comment:
K-G are not for string length, I don't believe.  They are used to truncate task hit and miss counts.